fig_test.go

```diff
@@ -249,6 +257,9 @@ func Test_fig_Load_Defaults(t *testing.T) {
 					Application struct {
 						BuildDate time.Time `fig:"build_date" default:"2020-01-01T12:00:00Z"`
 					}
+					Server struct {
+						Listener ListenerType `fig:"listener_type" default:"unix"`
```

Comment:
Do we need the enclosing `Server` struct?

Reply:
No. I'll remove it.